src/environment_setup/conda.ml

```diff
@@ -35,114 +63,49 @@ let installed ~(run_program : Machine.Make_fun.t) ~host ~install_path =
-https://repo.continuum.io/pkgs/free/linux-64/ncurses-5.9-0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/numexpr-2.4.6-np110py27_1.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/numpy-1.10.4-py27_1.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/openssl-1.0.2g-0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pandas-0.17.1-np110py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pip-8.0.3-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pixman-0.32.6-0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pycairo-1.10.0-py27_0.tar.bz2
-https://conda.anaconda.org/trung/linux-64/pyinstaller-3.1-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pyparsing-2.0.3-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pyqt-4.11.4-py27_1.tar.bz2
-https://conda.anaconda.org/bioconda/linux-64/pysam-0.9.0-py27_2.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pytables-3.2.2-np110py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/python-2.7.11-0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/python-dateutil-2.4.2-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pytz-2015.7-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/pyyaml-3.11-py27_1.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/qt-4.8.7-1.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/requests-2.9.1-py27_0.tar.bz2
-https://conda.anaconda.org/bioconda/linux-64/samtools-1.3-1.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/setuptools-20.1.1-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/sip-4.16.9-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/six-1.10.0-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/sqlite-3.9.2-0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/tk-8.5.18-0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/wheel-0.29.0-py27_0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/yaml-0.1.6-0.tar.bz2
-https://repo.continuum.io/pkgs/free/linux-64/zlib-1.2.8-0.tar.bz2|conda}
-
-(* Removed:
-   https://repo.continuum.io/pkgs/free/linux-64/readline-6.2-2.tar.bz2
-   This default conda compiled readline isn't linked appropriately:
-   symbol lookup error: /tmp/_MEIzf9vto/libreadline.so.6: undefined symbol: PC
-   We'll ignore it from the configuration and hope that the computer has a sane
-   readline lib installed by default.  *)
-
-let cfg_exists ~(run_program : Machine.Make_fun.t) ~host ~install_path =
+let configured ~conda_env ~(run_program : Machine.Make_fun.t) ~host =
   let open KEDSL in
-  let file = install_path // config in
-  let make =
-    run_program
-      ~requirements:[`Quick_run; `Self_identification ["conda"; "config-file"]]
-      Program.(exec ["mkdir"; "-p"; install_path]
-               && shf "echo %s >> %s" (Filename.quote biokepi_conda_config) file)
+  let create_env =
+    com ~conda_env "create -y -q --prefix %s python=%d" 
+      (envs_dir ~conda_env // conda_env.name)
```

Comment:
I think there is a problem with the `--prefix`. Maybe `envs_dir` is missing a `/conda_dir/` piece.

After success of the `Program.t` the env is not visible by conda:

```
$ /.../biokepi-work-dir/toolkit/python-tools/conda_dir/bin/conda env list
# conda environments:
#
root                     /.../biokepi-work-dir/toolkit/python-tools/conda_dir
```

The dir `/.../biokepi-work-dir/toolkit/python-tools/conda_dir/envs/` is empty whereas the `topiary` env was created one level above:

```
$ ls /.../biokepi-work-dir/toolkit/python-tools/envs/topiary/
bin  conda-meta  include  lib  share  ssl
```